app.js

```diff
@@ -5,92 +5,173 @@ var question2 = 'Question 2. Was I born in Washington State?';
 var question3 = 'Question 3. Have I ever coded before joining Code Fellows?';
 var question4 = 'Question 4. Have I ever been outside of the country?';
 var question5 = 'Question 5. Have I ever been to a live concert?';
+var question6 = 'What is my favorite number?';
+var question7 = 'Can you guess a band that I have seen live?';
 var correctResponse = 'Yep, That\'s right!';
 var incorrectResponse = 'Nope, that\'s not right';
+var favoriteNumber = 144;
+var bandsSeen = ['iron maiden', 'avenged sevonfold', 'rush', 'streetlight manifesto', 'slayer', 'metallica'];
+var correctAnswers = 0;
 
-alert('Hello there, this is my about me page. Were going to play a little game so that you can learn more about me. Please answer the following 5 questions in a yes or no fashion.');
-//the following block of code gets repeated five times (there are slight changes depending on whether or not the answer is true or false) I reckognize that this would work better as a function but since we haven't learned that yet I will just have the code coppied.
-var answer1 = prompt(question1);
-console.log('Asked the user the following question: ' + question1 + ' And they responded with ' + answer1);
-answer1 = answer1.toUpperCase();
-console.log('Changed their answer to ' + answer1 + ' in order to check if they provided a valid answer');
-if(answer1 === 'YES' || answer1 === 'Y'){
-  alert(incorrectResponse);
-  console.log('They got the wrong answer');
-}
-else if (answer1 === 'NO' || answer1 === 'N') {
-  alert(correctResponse);
-  console.log('They got the right answer');
-}
-else{
-  alert('It looks like your answer was not a yes or no response, please answer with a yes or no next time');
-  console.log('They answered improperly, the answer needed to be a yes or no');
-}
+var theirName = prompt('Why hello there, my name is Max, what\'s yours?');
+alert('Hi ' + theirName + ', nice to meet you. This is my about me page, before I let you take a look around I\'d like to ask you a few questions. Please respond to the first five with a yes or no, the rest will be self explanatory.');
 
-var answer2 = prompt(question2);
-console.log('Asked the user the following question: ' + question2 + ' And they responded with ' + answer2);
-answer2 = answer2.toUpperCase();
-console.log('Changed their answer to ' + answer2 + ' in order to check if they provided a valid answer');
-if(answer2 === 'YES' || answer2 === 'Y'){
-  alert(incorrectResponse);
-  console.log('They got the wrong answer');
+function askOne() {
+  var answer1 = prompt(question1);
+  console.log('Asked the user the following question: ' + question1 + ' And they responded with ' + answer1);
+  answer1 = answer1.toUpperCase();
```

Comment:
Here you are reassigning the value of the user's input which you're going to want to avoid in the future. Instead, you can .toUpperCase() in your conditionals like on line 24